cocos/2d/CCFontAtlas.cpp

```diff
@@ -415,6 +415,8 @@ bool FontAtlas::prepareLetterDefinitions(const std::u32string& utf32Text)
             tempDef.V = tempDef.V / scaleFactor;
         }
         else{
+            if(bitmap)
+             delete[] bitmap;
```

Comment:
May be you use `tab`, and its effect is different in different editors. Please use white space instead.